### PR TITLE
MAGEDOC-3672: Magento extensions

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Scope of this PR

- New Topic: Entirely new documents

## Purpose of this PR (required)

Add a new section for installing MDEs like Amazon Sales Channel and Google Shopping Ads Channel and move MSI and B2B topics there.

## Affected URLs (required)

- https://devdocs.magento.com/guides/v2.3/comp-mgr/install-extensions.html
- https://devdocs.magento.com/guides/v2.3/comp-mgr/install-extensions/inventory-management-installation.html
- https://devdocs.magento.com/guides/v2.3/comp-mgr/install-extensions/b2b-installation.html

whatsnew
Add a new [Magento Extensions](https://devdocs.magento.com/extensions/) section for installing Magento-developed extensions, such as Google Shopping ads Channel. Moved existing installation docs for B2B and MSI to new section.

### CODE REVIEW

**Comment by lorikrell** [comment]
I'll push updates to add info and make changes.

File: extensions/google-shopping-ads/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+group: extensions
+title: Google Shopping Ads Channel
+---
+
+The Google Shopping ads extension installs and adds Google Shopping features to Magento. To review additional information, see the Magento Marketplace page.
```
[the remark this answers is not in the document]
Comment (lorikrell) [reply extensions/google-shopping-ads/index.md]: It's ads. Google approved the name Google Shopping ads Channel.

File: extensions/google-shopping-ads/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+{:.bs-callout .bs-callout-info}
+Due to the technical nature of this installation, these instructions may require developer assistance.
+
+## Requirements
+
+- **Magento Instance**: Google Shopping Ads Channel can be installed on instances with Magento Open Source, Magento Commerce, and Magento Commerce Cloud versions, 2.2.X and 2.3.X. We do not support the extension on Magento 2.1 or Magento 1.
+- **Magento Web Account**: You should have a Magento web account, which is used to create and track an API key.
+- **API Key**: Get a Google Shopping ads API key through your Magento web account. The following instructions include these steps.
```
[the remark this answers is not in the document]
Comment (lorikrell) [reply extensions/google-shopping-ads/index.md]: Branding is ads, thanks though!

File: extensions/google-shopping-ads/index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- **Google accounts**: During onboarding, you will create and configure any required Google accounts and settings. Consider using email accounts for your business as the account will be the primary owner (admin) of the Google, Google Merchant Center and Google Ads accounts you set in this integration.
+  - Google Account: We recommend using a Google account associated to your business or company.
```
[the remark this answers is not in the document]
Comment (lorikrell) [reply extensions/google-shopping-ads/index.md]: Good point, editing!

File: extensions/amazon-sales/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+See [Installation]({{site.baseurl}}/extensions/install/).
+
+The name of the extension is `magento/module/amazon`.
+
+## Add the Google API key
+
+To add the Google API Key:
+
+1. On the Admin sidebar, tap **Marketing**. Then under Advertising Channels, tap **Google Shopping ads**.
```
[the remark this answers is not in the document]
Comment (lorikrell) [reply extensions/amazon-sales/index.md]: This is ported from the user guide which uses tap, swapping to click! thanks!

[the remark this answers is not in the document]
Comment (lorikrell) [reply extensions/amazon-sales/index.md]: Same on the bolding, it's user guide style, will update!

File: extensions/google-shopping-ads/index.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+    If you need an API Key, the following screen displays to walk through the process.
+
+    ![Onboarding API Key Verification](images/onboard-apikey-step1.png)
+
+1. Tap **Get API Key**. A new tab opens with [Magento Accounts](https://account.magento.com/customer/account/login) with the **Api Portal** tab open displaying the key.
+
+1. You may need to login with your Magento web account credentials. If you need to create an account, visit [here](https://account.magento.com/customer/account/login) and register. This account should be part of your company or business.
+
+1. API keys are available through the **Api Portal** tab. Tap  to copy your key.
```
[the remark this answers is not in the document]
Comment (lorikrell) [reply extensions/google-shopping-ads/index.md]: Image missing. thanks!

**Comment by lorikrell** [comment]
Thanks Kevin! Going through and updating! For anyone peeking into these, the Google and Amazon content is ported from User Guide. Giving it a cleaning currently for DevDocs styles. 👍

File: extensions/inventory-management/index.md [hunk 6]
```diff
@@ -1,27 +1,28 @@
...
 ---
 
 Inventory Management modules provide all inventory features and options for Single and Multi Source merchants to manage product quantities and stock for sales channels. These features are available in Magento 2.3.X Open Source, Commerce, and Commerce Cloud.
 
 These features and extensions were developed as part of the [Multi Source Inventory (MSI) project](https://github.com/magento-engcom/msi) through the Magento Community Engineering program.
 
-## Install Inventory Management
+## Install
 
 Inventory Management installs with Magento Open Source or Commerce with all features enabled by default. No additional steps are required for enabling these inventory features.
```
[the remark this answers is not in the document]
Comment (lorikrell) [reply extensions/inventory-management/index.md]: There is one, replacing!

File: extensions/b2b/index.md [hunk 7]
```diff
@@ -1,23 +1,26 @@
 ---
-group: software-update-guide
-title: Install the B2B extension
+group: extensions
+title: B2B
```
[the remark this answers is not in the document]
Comment (lorikrell) [reply extensions/b2b/index.md]: Edited to help out

File: extensions/google-shopping-ads/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (keharper) [reply extensions/google-shopping-ads/index.md]: Can we put [sic] after "ads"? (Kidding)

File: extensions/index.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+Magento extensions allow you to add custom features, functionality, services, and integrations to your Magento Admin and storefront. Extension developers package and install custom code as extensions, modules, components, addons, etc. You can design and develop extensions or purchase and install extensions from the Magento Marketplace or third-party providers.
+
+The [Magento Marketplace](https://marketplace.magento.com/) is the global eCommerce resource for applications and services that expand Magento solutions with powerful new features and functionality. Marketplace reviews and verifies all submitted extensions, modules, addons, and plugins are stable, supported, with clear documentation and release notes.
+
+This section tracks and provides installation guides and release notes for Magento developed and released packages.
+
+## Additional information
+
+* [Run the Module Manager]({{ page.baseurl }}/comp-mgr/module-man/compman-checklist.html)
```
Comment (jeff-matthews) [inline extensions/index.md]: @lorikrell, you can't use the `{{page.baseurl}}` jekyll variable in versionless directories. You must use absolute links.

File: extensions/index.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+* [Run the Extension Manager]({{ page.baseurl }}/comp-mgr/extens-man/extensman-checklist.html)
```
Comment (jeff-matthews) [inline extensions/index.md]: Must use absolute link

File: extensions/index.md (hunk 8 above)
Comment (lorikrell) [reply extensions/index.md]: Fixed!

File: extensions/index.md (hunk 9 above)
Comment (lorikrell) [reply extensions/index.md]: Fixed!

**Comment by lorikrell** [comment]
running tests

**Comment by lorikrell** [comment]
running tests

**Pull request merged**